components/resume/JobDetailsModal.tsx

```
@@ -0,0 +1,160 @@
+
+				<div className="flex justify-end pt-4 border-t">
+					<Button variant="outline" onClick={onClose}>
+						Close
+					</Button>
+				</div>
+			</DialogContent>
+		</Dialog>
+	)
+}
+
+export default JobDetailsModal
```

ESLint

[error] Newline required at end of file but not found.

(@stylistic/js/eol-last)